Lusas_Adapter/LusasAdapter.cs

```diff
@@ -190,6 +192,7 @@ public static bool IsApplicationRunning()
 
         private string m_directory;
         public double m_mergeTolerance = double.NaN;
+        public double g = 9.807;
```

Comment:
```suggestion
        public double m_g = 9.807;
```

The `m_` prefix is used for global objects - i.e. not defined within a method.